app/src/main/res/layout/fragment_login.xml

```diff
@@ -82,6 +82,14 @@
         android:text="@string/terms"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintTop_toBottomOf="@id/vSignUpButton"
-        app:layout_constraintEnd_toEndOf="parent"
-        />
+        app:layout_constraintEnd_toEndOf="parent" />
+
+    <ProgressBar
```

Comment:
You will need to move this to activities layout. Why? Because you can call it from other screens that you will need to create in the future